tools/utils.py

```diff
@@ -19,7 +19,7 @@
 # a helper class which implements the same version ordering as RPM
 class Version:
     def __init__(self, s: str) -> None:
-        self._s = s
```

Comment:
This change was because complaints about accessing a restricted member.

Reply:
This is pylint W0212: Access to a protected member _s of a client class (protected-access)

mypy doesn't care.

pylint has 45 other complaints about tools/import-wraps.py, and two complaints about W0212 protected-access for utils.py

Reply:
Reverted the change since it is unclear if pylint compliance is wanted.